fbw-a32nx/src/systems/fmgc/src/guidance/GuidanceController.ts

```diff
@@ -99,11 +95,11 @@ export class GuidanceController {
 
     automaticSequencing: boolean = true;
 
-    leftEfisState: EfisState
+    leftEfisState: EfisState<any>
```

Comment:
why EfisState<any>, and not EfisState<number>? 
Otherwise, LGTM.

Reply:
Or EfisState<unknown>.

Reply:
`unknown` isn't accepted by typescript here unfortunately